src/api/app/controllers/webui/users/notifications_controller.rb

```diff
@@ -67,6 +70,14 @@ def set_notifications
     @notifications = filter_notifications_by_kind(@notifications, @filter_kind)
   end
 
+  def set_notifications_count
+    @counted_notifications = {}
+    @counted_notifications['all'] = @notifications.count
```

Comment:
:thought_balloon: Isn't this `@notifications.unread.count + @notifications.read.count`?

Reply:
there is no default state...

Reply:
I don't understand what this discussion is about, sorry. Could you explain what do you mean exactly?
Btw yes, `@counted_notifications['all'] = @notifications.count` is the sum of `unread + read` ones.

Reply:
sorry I meant: there is no default state *in the query*